flake.nix

```diff
@@ -18,9 +18,32 @@
       owner = "NixOS";
       repo = "nixpkgs";
     };
+
+    # TODO: switch to main branch when merged
+    ogmios.url = "github:mlabs-haskell/ogmios/marton/nixos-module";
+
+    # TODO: clean this up when cardano-node is fixed
+    empty-flake.url =
+      "github:input-output-hk/empty-flake?rev=2040a05b67bf9a669ce17eca56beb14b4206a99a";
+    cardano-node-workbench = {
+      url =
+        "github:input-output-hk/cardano-node/ed9932c52aaa535b71f72a5b4cc0cecb3344a5a3";
+      inputs.membench.follows = "empty-flake";
+    };
+    cardano-node = {
+      url = "github:input-output-hk/cardano-node?ref=1.35.3";
+      inputs.cardano-node-workbench.follows = "cardano-node-workbench";
+      inputs.node-measured.follows = "cardano-node-workbench";
+    };
```

Comment:
As I've wrote in the other review there could be a better way.

Reply:
this will probably be fixed in next cardano-node release, otherwise we will try to contribute a fix upstream